napari/layers/labels/_labels_mouse_bindings.py

```diff
@@ -77,6 +77,8 @@ def draw(layer, event):
     undo_item = layer._undo_history[-1]
     if len(undo_item) == 1 and len(undo_item[0][0][0]) == 0:
         layer._undo_history.pop()
+    else:
+        layer.events.paint(value=undo_item)
```

Comment:
We need to also set the value in the `if` condition. Otherwise we might not populate the `value` of our `Event`

Reply:
hmmm that's interesting, the event shouldn't be firing at all in that case, it's weird that it fires but has no `value`... Let me investigate